packages/babel-traverse/src/scope/index.js

```diff
@@ -593,7 +593,7 @@ export default class Scope {
       if (node.computed && !this.isPure(node.key, constantsOnly)) return false;
       if (node.kind === "get" || node.kind === "set") return false;
       return true;
-    } else if (t.isClassProperty(node)) {
+    } else if (t.isClassProperty(node) || t.isObjectProperty(node)) {
```

Comment:
But thats a `ClassMethod` not class property which is stage 1 https://github.com/jeffmo/es-class-fields-and-static-properties - although even that apparently has it? 

http://astexplorer.net/#/xSb5f2rOJy/1